google-cloud-spanner/pom.xml

```diff
@@ -49,6 +51,7 @@
             <id>default-test</id>
             <configuration>
               <excludedGroups>com.google.cloud.spanner.TracerTest,com.google.cloud.spanner.IntegrationTest</excludedGroups>
+              <skipTests>${skipUTs}</skipTests>
```

Comment:
I only see this being set to `false`. Do we need this configuration?

Reply:
Hi Jeff, I added this configuration just for our internal test to be able to use `-DskipUTs=true`. Currently we are using `mvn verify -am -pl google-cloud-spanner -B -Penable-integration-tests,spanner-directpath-it -DskipUTs=true -D... ...` for our internal E2E DP tests, and meanwhile I don't want to change the behavior for other tests. Do you think this is the right way for approaching this purpose?

Reply:
I might add a note somewhere so we don't inadvertently break this.

We might also want to add something like this to our shared setup.